packages/graphql/README.md

```diff
@@ -43,20 +39,53 @@ This argument can be specified multiple times to add multiple HTTP headers.
+- [Mock union types](https://github.com/xing/hops/tree/refactor-graphql-mock-server/packages/spec/integration/graphql-mock-server/mocks/exercise6)
+- [Mock enums](https://github.com/xing/hops/tree/refactor-graphql-mock-server/packages/spec/integration/graphql-mock-server/mocks/exercise7)
+- [Mock mutations with success and error fields](https://github.com/xing/hops/tree/refactor-graphql-mock-server/packages/spec/integration/graphql-mock-server/mocks/exercise8)
+- [Cursor-based pagination with connection and edges](https://github.com/xing/hops/tree/refactor-graphql-mock-server/packages/spec/integration/graphql-mock-server/mocks/exercise9)
+
 ### Configuration
 
 #### Preset Options
 
-| Name                     | Type      | Default                        | Required | Description                                                              |
-| ------------------------ | --------- | ------------------------------ | -------- | ------------------------------------------------------------------------ |
-| `fragmentsFile`          | `String`  | `<rootDir>/fragmentTypes.json` | _no_     | Where to store the generated fragment types file                         |
-| `graphqlSchemaFile`      | `String`  | `''`                           | _no_     | Path to your GraphQL schema file                                         |
-| `graphqlUri`             | `String`  | `''`                           | _yes_    | Url to your GraphQL endpoint                                             |
-| `shouldPrefetchOnServer` | `Boolean` | `true`                         | _no_     | Whether Hops should execute GraphQL queries during server-side rendering |
+| Name                      | Type      | Default                                 | Required | Description                                                              |
+| ------------------------- | --------- | --------------------------------------- | -------- | ------------------------------------------------------------------------ |
+| `fragmentsFile`           | `String`  | `<rootDir>/fragmentTypes.json`          | _no_     | Where to store the generated fragment types file                         |
+| `graphqlSchemaFile`       | `String`  | `''`                                    | _no_     | Path to your GraphQL schema file                                         |
+| `graphqlUri`              | `String`  | `''`                                    | _yes_    | Url to your GraphQL endpoint                                             |
+| `shouldPrefetchOnServer`  | `Boolean` | `true`                                  | _no_     | Whether Hops should execute GraphQL queries during server-side rendering |
+| `enableGraphqlMockServer` | `Boolean` | `process.env.NODE_ENV !== 'production'` | _no_     | Whether Hops should start an Apollo GraphQL Mock Server                  |
```

Comment:
I think this should be an environment variable, the default could still be `process.env.NODE_ENV !== 'production'` though.

We have (mostly / at least) 4 kinds of configurations:

1. True isomorphic configuration or configuration that is needed on the client.
2. Configuration that is only needed on the server but doesn't hurt to be on the client as well.
3. Environment dependent configuration.
4. Sensitive informations like tokens and secrets.

And I would like to use environment variables for type 3 and 4 and in my opinion `enableGraphqlMockServer` is type 3.